### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by gmao-rreichle**
- removed doc/GEOSldas-tutorial.pptx
- added mark-down document on GEOSldas
  - Output Specs
  - Configuration Files
- updated CHANGELOG.md with latest release (v17.9.1)
- documentation updates in ldas_setup (re. batinp.txt file)
- minor clean-up in README.md and GEOSldas_LDAS.rc
- revised -debug option in lenk.j

### CODE REVIEW

**Comment by weiyuan-jiang** [comment]
I have updated the debug flag in lenkf.k.template. Now users should use lenkf.j -debug instead of lenkf.j 1 or lenkf.j 2 . For the first time when users use lenkf.j -debug,  instructions are printed out without launching the debugging tools.

File: doc/README.ConfigurationFiles.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+---
+### `CAP.rc` and `cap_restart`
+
+Created by `ldas_setup`.
+
+`CAP.rc` contains the experiment start/end times and the number/length of job segments. See documentation in `"exeinp"` file.
+
+`cap_restart` contains the start time of the next job segment.  `GEOSldas.x` reads the start time from this file. [_**CORRECT???**_]
```
Comment (gmao-rreichle) [inline doc/README.ConfigurationFiles.md:41]: @weiyuan-jiang : Is the 2nd sentence in this line correct?

File: doc/README.OutputSpecs.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ tavg3_2d_lnd_Nx.mode:        'time-averaged',
+ tavg3_2d_lnd_Nx.frequency:   030000,
+ tavg3_2d_lnd_Nx.fields:      'WCSF'       , 'CATCH'  , 'SFMC'         ,
+                              'WCRZ'       , 'CATCH'  , 'RZMC'         ,
+                              ::
+```
+
+The same variable can be written in more than one Collection (i.e., at different temporal and/or spatial resolutions), and there is no limit to the number of Collections that are defined for and written by a simulation.  
+
+Gridded ("2d") output can be on a grid other than the "native" grid that is associated with the tile space used in the simulation, as long as the output grid is defined in `HISTORY.rc`.   [_**MORE INFO NEEDED!!!**_]
```
Comment (gmao-rreichle) [inline doc/README.OutputSpecs.md:54]: @weiyuan-jiang : I never quite understood how exactly the output grid is defined in HISTORY.rc and how it is specified for a given Collection.  As it stands, this section pretty much ignores this aspect, but we'll need to add more info eventually.

File: doc/README.ConfigurationFiles.md (hunk 1 above)
Comment (weiyuan-jiang) [reply doc/README.ConfigurationFiles.md:41]: Yes, it is correct. The start time is from the file "cap_restart"

File: doc/README.OutputSpecs.md (hunk 2 above)
Comment (weiyuan-jiang) [reply doc/README.OutputSpecs.md]: I have added more info to it.

**Comment by weiyuan-jiang** [comment]
looks good to me.

**Review by biljanaorescanin — APPROVED** [review]

**Pull request merged**